tests/jupyter/test_pythreejs.py

```diff
@@ -214,3 +224,20 @@ def test_labels():
     # ensure that we ignore labels
     with pytest.warns(UserWarning):
         pv_pythreejs.convert_plotter(pl)
+
+
+def test_linked_views(sphere):
+
+    n_row, n_col = (2, 3)
+    pl = pyvista.Plotter(shape=(n_row, n_col))
+
+    for ii in range(n_row):
+        for jj in range(n_col):
+            pl.subplot(ii, jj)
+            pl.add_mesh(sphere)
+    pl.link_views()
```

Comment:
We don't necessarily have to write a test for this, but at least once you should check that linking works in the right direction. What I mean is that a linear-to-2d index mapping always has a potential ambiguity of row-major vs column-major order, and if we get it wrong then we'd be linking the wrong cameras. So something like linking `(0, 0)` with `(1, 2)` in a grid sized `(3, 4)` would probably settle this. Even if just to test it locally once.

Reply:
Implemented in bde4bab15e5ec0f2c5ed72fc45084c64a40e3b40.